docs/use-cases/hydra-for-payments/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Hydra for Payments",
```

Comment:
Do we really want to repeat "Hydra for" in each use case section?

If we avoid that, we can easily keep the `inter-wallet-payments` and `pay-per-use-api` documents in the "Payments" category until we have something more structured.

Could: Rename to "Payments" and `use-cases/payments/` accordingly, and move the payment use case documents beneath this directory